Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -1765,147 +1634,147 @@ public static void LookupPlayerNamesByPlayerPublicUIds(string[] playerPublicUIds
                 return;
             }
 
-            LootLockerAPIManager.LookupPlayerNames(new LookupPlayerNamesRequest()
-            {
-                player_public_uids = playerPublicUIds
-            }, onComplete);
+            LootLockerAPIManager.LookupPlayerNames("player_public_uid", playerPublicUIds, onComplete);
         }
 
         /// <summary>
-        /// Get player names of the players from their last active platform by steam ID's. You can read more on how to setup Steam with LootLocker here; https://docs.lootlocker.com/how-to/authentication/steam
+        /// Get player names and important ids of a set of players from their last active platform by public playerID's.
```

Comment:
On this one and the remaining LookUp-functions, the wrong ID is mentioned.
This for example should say:
`last active platform by player ulids.`

Reply:
Upon further inspection it's this one, LookupPlayerNamesByPlayerUlids, LookupPlayerNamesByGuestLoginIds and LookupPlayerNamesByPlayerNames.
So not all, but these ones.

Reply:
Fixed